client/layout/masterbar/woo.scss

```diff
@@ -873,5 +873,39 @@
 				display: none;
 			}
 		}
+
+		.spinner__outer,
+		.spinner__inner {
+			border-top-color: var(--wp-admin-theme-color);
+		}
+		.spinner__inner {
+			border-right-color: var(--wp-admin-theme-color);
+		}
+
+		// Lost password
+		.login__lostpassword-form {
+			display: flex;
+			flex-direction: column;
+			padding: 54px 24px 32px;
```

Comment:
It seems that the padding-top should be set to `48px`?

![Screenshot 2023-06-20 at 11 45 44](https://github.com/Automattic/wp-calypso/assets/4344253/5d017d7b-f59b-4678-b00b-c780320f9fdd)

Reply:
Good catch! Fixed in 7352c79420